doc/source/_static/ensight_rest_v1.yaml

```diff
@@ -0,0 +1,594 @@
+openapi: 3.0.0
```

Comment:
I'm not sure we want two copies of the openapi file.  Should this not be a copy command in the yaml script?  Just before building the docs, copy the file over into the static directory?

Reply:
@randallfrank the problem is that when I cleaned the old "dev" and "release" folders in gh-pages, this was the only one with a broken link. I have placed manually the file on the new gh-pages folders, but copying it should work as well. I am looking at the codecov issue right now but I can change it for sure

Reply:
I believe I added the copy step